includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -203,11 +203,30 @@ public function __construct(
 		$this->order_service                        = $order_service;
 		$this->duplicate_payment_prevention_service = $duplicate_payment_prevention_service;
 
-		$this->id                 = static::GATEWAY_ID;
-		$this->icon               = plugins_url( 'assets/images/payment-methods/cc.svg', WCPAY_PLUGIN_FILE );
-		$this->has_fields         = true;
-		$this->method_title       = 'WooPayments';
-		$this->method_description = WC_Payments_Utils::esc_interpolated_html(
+		$this->id           = static::GATEWAY_ID;
+		$this->icon         = plugins_url( 'assets/images/payment-methods/cc.svg', WCPAY_PLUGIN_FILE );
+		$this->has_fields   = true;
+		$this->method_title = 'WooPayments';
+
+		$description_links = [
```

Comment:
Should we move these to a function `get_description` to keep the constructor clean?

Reply:
Fixed on https://github.com/Automattic/woocommerce-payments/pull/6899/commits/8d5d12ac85390d7ad240e8e5f109a688956fa284.